eftest/src/eftest/runner.clj

```diff
@@ -16,24 +16,44 @@
   (or (-> v meta :eftest/synchronized true?)
       (-> v meta :ns meta :eftest/synchronized true?)))
 
+(defn- known-slow? [v]
+  (or (-> v meta :eftest/slow true?)
+      (-> v meta :ns meta :eftest/slow true?)))
+
 (defn- failed-test? []
   (or (< 0 (:error @test/*report-counters* 0))
       (< 0 (:fail @test/*report-counters* 0))))
 
+(defn- run-test-and-report-time
+  [test-warn-length start-time v]
+  (let [result   (test/test-var v)
+        end-time (System/nanoTime)
+        duration (/ (- end-time start-time) 1e6)]
+    (when (and (not (known-slow? v))
+               (number? test-warn-length)
+               (<= test-warn-length duration))
+      (binding [clojure.test/*testing-vars* [v]]
+        (test/report {:type     :long-test
+                      :duration duration
+                      :var      v})))
+    result))
+
 (defn- test-vars
-  [ns vars {:as opts :keys [fail-fast? capture-output?] :or {capture-output? true}}]
-  (let [once-fixtures (-> ns meta ::test/once-fixtures test/join-fixtures)
+  [ns vars {:as opts :keys [fail-fast? capture-output? test-warn-length] :or {capture-output? true}}]
```

Comment:
Can you ensure that lines are wrapped at 80 characters?

Reply:
done